### PR TITLE
Badge on cover (book-detail-4)

### PR DESCRIPTION
**Pull request by dsandoval11**
## Summary
- Badge added on the book cover

## Screenshot
image

## Trello Card
https://trello.com/c/rqsCx6pd/29-detalle-de-libro-iv-badge

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,12 +10,9 @@
...
   <body>
     <div class="book">
-      <div>
-        <img
-          class="book-cover"
-          src="./images/book-cover.png"
-          alt="book-cover"
-        />
+      <div class="book-cover">
+        <img class="cover" src="./images/book-cover.png" alt="book-cover" />
```
Comment (mcallegari10) [inline index.html:14]: Alt should be an explanation of the image, in this case, could be "Cook cover". No need for any middle-scores 😄

File: scss/detail.scss [hunk 2]
```diff
@@ -7,11 +7,24 @@
...
+    max-width: 261px;
+    width: 100%;
+  }
+
+  .badge {
+    height: 100%;
+    max-height: 104px;
+    position: absolute;
+    right: 32px;
+    top: -16px;
```
Comment (mcallegari10) [inline scss/detail.scss:25]: Try to use bottom instead, we avoid the use of negative numbers for absolute positions

File: scss/styles.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+body {
+  padding: 32px;
```
Comment (mcallegari10) [inline scss/styles.scss:2]: Are you sure you need this in the body? Maybe the view container is a better suit 😄

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (dsandoval11) [reply index.html:14]: Fixed in PR #6

File: scss/detail.scss (hunk 2 above)
Comment (dsandoval11) [reply scss/detail.scss:25]: Fixed in PR #6

File: scss/styles.scss (hunk 3 above)
Comment (dsandoval11) [reply scss/styles.scss:2]: Fixed in PR #5

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**